src/Yolo.java

```diff
@@ -94,42 +92,38 @@ private static void analyzeFrame(VideoCapture cap, Mat frame, Net net) {
 		int height;
 		int left;
 		int top;
+		Mat row;
+		Mat scores;
+		Mat level;
+		Core.MinMaxLocResult mm;
+		float confidence;
+		Point classIdPoint;
 		
-		/*clsIds.clear();
-		confs.clear();
-		rects.clear();*/
-
-		Mat blob = Dnn.blobFromImage(frame, 0.00392, sz, new Scalar(0), true); 
+		Mat blob = Dnn.blobFromImage(frame, 0.00392, sz, new Scalar(0), true);  //edit this maybe, scalar is empty rn, so no mean subtraction i think
```

Comment:
constants should be defined for values by conventions.

Reply:
I just changed some free floating numbers to class constants.